TemplateBicep/modules/alerts.bicep

```diff
@@ -0,0 +1,271 @@
+param iothubName string
+param appInsightsName string
+
+@description('Dynamic Threshold sensitivity (High, Medium, Low)')
+param alertSensitivity string = 'Medium'
```

Comment:
nit: I think all below could be defined as variables as the main template has no option to change them

Reply:
I will leave it for now. If we ever want to make them customizable, we only need to enable them on the main.bicep.